### ISSUE TITLE
[Feature Request] - Create new /setup page

### ISSUE THREAD

**Issue by johnkahn** [AmericanAirlines/Hangar#67]
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Is your feature request related to a problem, or an open issue? Describe the problem, and if applicable, reference the issue.
On navigating to `/setup` the user should see a page which includes a list of default config items. Each config item should be editable/toggle-able, and all should have a value before the user is allowed to proceed. 

After editing/setting values, the user should see a button to "Complete Setup", which will then redirect them back to the homepage.

This page should utilize the Config component created by #295 

Dependent on #297
Dependent on #299

**Comment by SpencerKaiser** [comment]
Closing in favor of #331
